cybershell/bypass_techniques.py

```
@@ -0,0 +1,620 @@
+                    "path": path,
+                    "headers": headers.copy(),
+                    "params": params.copy(),
+                    "bypass_technique": f"method_{bypass_method}",
+                    "success_rate": 0.3
+                }
+                bypass_requests.append(request)
+        
+        # Parameter pollution
+        if params:
+            polluted_params = params.copy()
+            for key in params.keys():
```

Ruff (0.12.2)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)